assets/javascripts/discourse/models/docs.js

```diff
@@ -53,6 +57,12 @@ Docs.reopenClass({
       return data;
     });
   },
+
+  _initUserModels(post) {
+    if (post.mentioned_users) {
+      post.mentioned_users = post.mentioned_users.map((u) => User.create(u));
+    }
+  },
```

Comment:
We just need to make sure users data are wrapped into user models to make status live.